docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml

```diff
@@ -58,9 +58,10 @@ content:
   field_rating:
     weight: 3
     label: above
-    settings: {  }
+    settings:
+      fivestar_widget: modules/fivestar/widgets/basic/basic.css
     third_party_settings: {  }
-    type: fivestar_rating
+    type: fivestar_stars
```

Comment:
yeah! I recreated the "Resource" feature from dev. It had this much changes.
Maybe somebody directly modified in dev, and did not exported as feature.
Or change made in code and not synced in dev!